.github/workflows/esphome-parallel.yaml

```
@@ -20,22 +19,25 @@ jobs:
+      file: ${{ steps.set-files.outputs.file }}
     steps:
       - name: Getting your configuration from GitHub
         uses: actions/checkout@v4
       - name: Get list of files
-        id: set-files  # Give it an id to handle to get step outputs in the outputs key above
-        run: echo "::set-output name=file::$(ls esphome/*.yaml | jq -R -s -c 'split("\n")[:-1]')"
-        # Define step output named file base on ls command transformed to JSON thanks to jq
+        # ID is referenced by outputs above
+        id: set-files
+        # Use jq to create a named file list
+        run: echo "::set-output name=file::$(ls esphome/*.yaml | jq -R -s -c 'split("\n")[:-1]')"  # yamllint disable-line rule:line-length
```

actionlint (1.7.7)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)